src/v1/media.js

```diff
@@ -106,6 +106,8 @@ export class Media extends Resource {
           usertags: JSON.stringify(userTags),
           _uid: accountId.toString(),
           device: session.device.payload,
+          geotag_enabled: true,
```

Comment:
Why is this always true?

Reply:
I had checked the web request and contained this parameter, I will make a change in this commit to leave this value true only when there is a location.

Reply:
Will wait for it